src/Advection/vector_invariant_velocity_upwinding.jl

```diff
@@ -0,0 +1,97 @@
+const VectorInvariantVelocityVerticalUpwinding  = VectorInvariant{<:Any, <:Any, <:Any, <:Any, <:AbstractUpwindBiasedAdvectionScheme, <:VelocityUpwinding}
+
+#####
+##### Velocity upwinding is a Partial Upwinding where the upwind choice occurrs _inside_
+##### the difference operator (i.e velocity upwinding) instead of outside (i.e., derivative upwinding).
+##### _MOST_ stable formulation at the expense of a low kinetic energy
+##### 
+
+##### 
+##### Velocity Upwinding of Divergence flux
+#####
+
+@inline function Ax_uᶜᶜᶜ(i, j, k, grid, scheme, u) 
```

Comment:
This `Ax_uᶜᶜᶜ` is quite easy to confuse with `Ax_qᶠᶜᶜ`. Can we make it more explicit that A*u has been reconstructed at ccc?

Reply:
(same hold for everything)

Reply:
there is `ccc` at the top no? It is written like that exactly to show the similarity with `Ax_qᶠᶜᶜ`.

Reply:
Yes, there is the ccc... but it'a a bit cryptic (like one can miss it).
Perhaps `interpolated_ Ax_uᶜᶜᶜ` ? or `reconstructed_ Ax_uᶜᶜᶜ`?